pkg/ebpfev/event_interface.go

```diff
@@ -0,0 +1,14 @@
+package ebpfev
+
+import "time"
+
+type EventData struct {
+	Timestamp   time.Time
```

Comment:
[Required]: Why are the fields public?

Reply:
cause I use this fields out of this package and I don't want to create getters and setters for any field

Reply:
done